charts/gateway/README.md

```diff
@@ -1157,6 +1158,50 @@ The full default is this
     com.l7tech.server.clusterStaleNodeCleanupTimeoutSeconds=86400
     # Additional properties go here
 ```
+### DISKLESS Configuraton
+DISKLESS_CONFIG is a flag to pass gateway sensitive data to be mounted to the container gateway file system or passed as environment variables.
+By Default, true, environment variables are used to configure Gateway.
+
+When DISKLESS_CONFIG is disabled , Gateway will be configured from node.properties
+
+#### node.properties
+Note: 
+- Database configuration should be same as mentioned in node.properties
+```
+node.cluster.pass=newpassword
+admin.user=admin
+admin.pass=newpassword
+node.db.config.main.host=<Replace with jdbc-url>
```

Comment:
jdbc url is composed of db host, db port, and db name

just provide some hostname here like "myhost.com"